src/WTG.Analyzers/Rules/Rules.xml

```diff
@@ -231,6 +231,11 @@
 			<description>Combining bool expressions in larger boolean expressions makes the code more complex than it needs to be.</description>
 			<tag>Unnecessary</tag>
 		</rule>
+		<rule id="16" name="InterpolatedStringMustBePurposeful" severity="Warning">
```

Comment:
Perhaps this should just be default of Info, its not that bad.

Reply:
You will need to add it to warnall though - https://github.com/WiseTechGlobal/WTG.Analyzers/blob/master/src/WTG.Analyzers/build/WarnAll.editorconfig